src/renderer/components/wallet/account/AccountSelector.styles.ts

```diff
@@ -52,5 +52,5 @@ export const Label = styled(UILabel).attrs({
 `
 
 export const WalletTypeLabel = styled(WalletTypeLabelUI)`
-  margin: -45px 0px 0px 10px;
+  margin: -20px 0px 0px 10px;
```

Comment:
This change will effect SendView as well (as discussed in DM) - will be fixed with another PR

![Screenshot from 2021-09-30 11-04-17](https://user-images.githubusercontent.com/61792675/135423148-a60b50cd-939e-4767-bf12-dc9b912e2a0c.png)